examples/interaction/js_callbacks/setvalue.py

```diff
@@ -1,3 +1,14 @@
+''' An visualization of buttons in bokeh.models. This example demonstrates
+changing the value of an object when a certain event(like clicking of a button)
+occurs through SetValue.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.Button, Bokeh.models.SetValue
+    :refs: :ref:`ug_interaction_js_callbacks_SetValue`
```

Comment:
Sphinx is picky and case-sensitive about these things (all our manual sphinx ref anchort are all lower case)
```suggestion
    :refs: :ref:`ug_interaction_js_callbacks_setvalue`
```